### PR TITLE
Additional info about user's EventsSeenUpTo in Start/ResumeChat and S…

### PR DESCRIPTION
**Pull request by gluszczyk**
…endEvent.

## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7061-activate-chat-seen-events)
- [Jira](https://livechatinc.atlassian.net/browse/API-6987)

## 📓 Description

Start/ResumeChat - describes new behavior in v3.3
SendEvent - adds missing info

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

File: content/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -1506,10 +1506,12 @@ https://api.livechatinc.com/v3.2/agent/action/remove_user_from_chat \
 
 ### Send Event
 
-Sends an [Event](#events) object. Use this method to send a message by specifing the [Message](#message) event type in the request.
+Sends an [Event](#events) object. Use this method to send a message by specifying the [Message](#message) event type in the request.
 
 It's possible to write to a chat without joining it. The user sending an event will be automatically added to the chat with the `present` parameter set to `false` (see [`user_added_to_chat`](/messaging/agent-chat-api/rtm-reference/#user_added_to_chat)). **Sample use case:** a Bot Agent that sends messages visible to all chat participants without actually joining the chat.
 
+Updates requester's `events_seen_up_to` as if they've seen all chat events.
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/index.mdx:1513]: + Please, apply the same change in other documents.

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -18,6 +18,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **Unfollow Chat** ([Web](/messaging/agent-chat-api/v3.3/#unfollow-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#unfollow-chat)) method had the `chat_id` parameter renamed to `id`.
 - The **Activate Chat** method was renamed to **Resume Chat** ([Web](/messaging/agent-chat-api/v3.3/#resume-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#resume-chat)).
 - The **Start Chat** ([Web](/messaging/agent-chat-api/v3.3/#start-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#start-chat)) and **Resume Chat** ([Web](/messaging/agent-chat-api/v3.3/#resume-chat) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#resume-chat)) methods have a new optional parameter, `active`.
+- The **Start Chat** and **Activate Chat** methods now update requester's `events_seen_up_to` as if they've seen all chat events. Subsequent call to `mark_events_as_seen` can be skipped in these cases.
```
[the remark this answers is not in the document]
Comment (gluszczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:21]: and now we have `the requester's` almost next to each other, kinda bothers me 😉

[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:21]: Well, what can I say? :D I believe we can compromise the style for the sake of clarity. 

How about?

> The **Start Chat** and **Activate Chat** methods now update the requester's `events_seen_up_to` as if all chat events were already seen by the requester.

[the remark this answers is not in the document]
Comment (gluszczyk) [reply content/messaging/agent-chat-api/changelog/index.mdx:21]: ok, that's an acceptable compromise :trollface:

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**